packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -0,0 +1,119 @@
+import './index.scss';
+import { Desktop, Pattern, Touch } from '@carbon/pictograms-react';
+import { object, select, text, withKnobs } from '@storybook/addon-knobs';
+import ContentGroupPictograms from '../ContentGroupPictograms';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Blocks)|Content Group Pictograms', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const pictograms = {
+      Desktop: 'Desktop',
+      Touch: 'Touch',
+      Pattern: 'Pattern',
+    };
+
+    const pictogram1 = select(
+      'Element 1 pictogram (required)',
```

Comment:
maybe we can keep our language consistent. use Item instead of Element, or even just nix Element altogether.